<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="F198DTRV" ignored="0" name="TC2" tests="1" failures="0" timestamp="30 Jun 2017 12:16:26 GMT" time="5.04" errors="0">
  <testcase name="@BeforeClass setUp" time="0.0" classname="com.nscomp.tests.VerifySignIn">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.nscomp.setup.DriverSetup.getDriver(DriverSetup.java:33)
	at com.nscomp.tests.VerifySignIn.setUp(VerifySignIn.java:45)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeClass setUp -->
  <testcase name="@AfterClass tearDown" time="0.0" classname="com.nscomp.setup.DriverSetup">
    <skipped/>
  </testcase> <!-- @AfterClass tearDown -->
  <testcase name="signInInvalidEmail" time="0.0" classname="com.nscomp.tests.VerifySignIn">
    <skipped/>
  </testcase> <!-- signInInvalidEmail -->
</testsuite> <!-- TC2 -->
